package.json

```diff
@@ -25,8 +25,8 @@
     "typescript": "^4.3.5"
   },
   "devDependencies": {
-    "@mate-academy/scripts": "^0.9.2",
     "@mate-academy/eslint-config": "*",
+    "@mate-academy/scripts": "^1.2.8",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use `git add src/.` & `git commit -m 'your message'`